### ISSUE TITLE
[Feature] Generic CSV upload

### ISSUE THREAD

**Issue by zcrt** [minvws/nl-kat-rocky#151]
## Problem description
Uploading objects through a `.csv`-file is restricted to four specific object classes (`URL`, `IPv4Address`, `IPv6Address`, `Hostname`). This requires multiple file uploads and does not cover other objects such as `Network` or `NetBlock`.

## Describe the solution you'd like
It would be nice if object types could be specified directly in the `.csv`-file and not be restricted to specific classes. (E.g. it is not possible to upload a `NetBlock` or `Network` object now.)

## Describe alternatives you've considered
- Connect an external database, this is not supported yet.
- Manually add individual objects, not really an improvement, especially when the KATdatabase is cleared (e.g. unintentionally).

**Comment by underdarknl** [comment]
Agreed. uploads should handle all OOI Types. A normalizer, and schema for the specific layouts is part of 1.6.
Mixing multiple types in one CSV is unexpected though, as that would require more than one header line to specify the column names. Maybe an excel file (with sheets) or a zip file (with multiple csv's) would work.

**Comment by zcrt** [comment]
Great. Regarding the `.csv`: alternatively the first column just defines the object type,
